examples/lang/env0.mcl

```diff
@@ -4,19 +4,19 @@
 import "fmt"
 import "sys"
 
-$x = sys.getenv("TEST", "321")
+$x = sys.defaultenv("TEST", "321")
 
 print "print1" {
 	msg => fmt.printf("the value of the environment variable TEST is: %s", $x),
```

Comment:
I suspect this message alludes to this being a regular "getenv" call... So it has one too many args.

Reply:
Updated.